deployments/examples/service_policies/policies/proxy.rego

```diff
@@ -3,10 +3,16 @@ package proxy
 import future.keywords.if
 import data.utils
 
-default granted = true
+default granted := true
 
-granted := false if {
+granted = false if {
     utils.is_request_type_put
     not input.request.path == "/data"
     not utils.collection_contains(utils.ALLOWED_FILE_EXTENSIONS, input.request.path)
 }
+
+granted = false if {
+    utils.is_request_type_post
+    startswith(input.request.path, "/remote.php")
```

Comment:
This seems wrong. I can (and should) omit the `remote.php` part when I send requests to ocis. What is the point of it?

Reply:
There is another POST request for the login  which should not be blocked by this rule.

Reply:
I don't understand. My point was that checking for `remote.php` doesn't bring any value. Either requests of a client will start with `remote.php` or none will. It is even possible that both is true. So I don't understand why we check for it in the policies?

Reply:
Ja ok fine. Just the example makes no sense. But it is not critical. We can go with it.

Reply:
`POST - https://localhost:9200/konnect/v1/token`
`POST - https://localhost:9200/api/v0/settings/roles-list`
`POST - https://localhost:9200/api/v0/settings/roles-list`

should not be blocked.... use urls to identify the type is not the best solution, i agree.

i can hydrate the environment if it's a upload and or create request. We can iterate on it.

Reply:
I was just about the `remote.php` part. This makes no sense as I can send all requests also without it and would still get the same results. But you are right, let's iterate over it when we see fit.